src/client/components/TotalPriceCard/TotalPriceCard.component.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TotalPriceCard.styles.css';
+
+const TotalPrice = ({ quantity, price }) => {
```

Comment:
Hey, Bikram. We was talk with you about this component but I missed something here. Because we don't need quantity in this component. Because this component gets the total price from the cart page as a prop. You can check the Figma cart page for this situation

Reply:
yes , I just put them in here to mock the controls of the inputs, we can instead give the subTotal value which we get from another component fx add to cart.